qp_klp/tests/data/configuration_profiles/default.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Why do we need this and the other json here? I thought they could be taken from mg-scripts  ... no?

Reply:
no. these are to support the unittests, which require initializing a pipeline object and hence a sample configuration file and configuration_profiles directory with sample profiles.